### PR TITLE
Deprecate unused thumb_url and supporting methods

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: lib/dor/models/concerns/publishable.rb [hunk 1]
```diff
@@ -39,6 +41,7 @@ def thumb
 
        thumb_image
     end
+    deprecation_deprecate :thumb
```
Comment (peetucket) [inline lib/dor/models/concerns/publishable.rb]: `thumb` is still used in generating public XML here: https://github.com/sul-dlss/dor-services/blob/master/lib/dor/services/public_xml_service.rb#L25

Comment (peetucket) [reply lib/dor/models/concerns/publishable.rb]: originally here when first implemented: https://github.com/sul-dlss/dor-services/pull/237/files#diff-831fbf194eaba823278738aa38837ceaR100 (e

**Comment by peetucket** [comment]
If `#thumb` is still being used in the public XML, doesn't leaving it mark as deprecated risk that some future developer will inadvertently remove it without realizing this?

**Comment by jcoyne** [comment]
@peetucket I updated `PublicXmlService` to use the new`ThumbnailService` and not `object.thumb`

**Comment by peetucket** [comment]
right, never mind, i see now the extracted service method is not deprecated 👍

**Pull request merged**